build-apisix-runtime.sh

```diff
@@ -158,6 +158,12 @@ grpc_engine_path="-DNGX_GRPC_CLI_ENGINE_PATH=$OR_PREFIX/libgrpc_engine.so -DNGX_
 cd openresty-${OPENRESTY_VERSION} || exit 1
 
 or_limit_ver=0.08
+enable_http3=${enable_http3:-}
```

Comment:
why do we need to support `enable_http3` as an input flag?

enable http3 as default, I think it is acceptable

Reply:
as we discussed by online meeting, I think we can remove `if ... fi`

the next APISIX will contain http3 feature